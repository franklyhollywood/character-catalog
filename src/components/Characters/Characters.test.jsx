import { MemoryRouter } from 'react-router-dom';
import { render } from '@testing-library/react';
import Characters from './Characters.jsx';

it('renders the characters view', () => {
  const characters = [
    {
      image: 'someurl',
      name: 'fred',
      species: 'ape',
      status: 'dead',
    },
  ];

  const { container } = render(
    <MemoryRouter>
      <Characters characters={characters} />
    </MemoryRouter>
  );

  expect(container).toMatchSnapshot();
});
