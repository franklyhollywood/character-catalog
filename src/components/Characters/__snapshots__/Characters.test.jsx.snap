// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the characters view 1`] = `
<div>
  <ul
    aria-label="characters"
    class="list"
  >
    <li
      class="item"
    >
      <a
        href="/characters/undefined"
      >
        <figure>
          <img
            alt="fred"
            class="image"
            src="someurl"
          />
          <figcaption
            class="linkText"
          >
            fred
          </figcaption>
        </figure>
      </a>
    </li>
  </ul>
</div>
`;
